examples/MOTD/run.py

```diff
@@ -20,7 +20,7 @@
 
 # Do whatever you want to do with log messages! Here we create a log file.
 logger = logging.getLogger()
-logger.addHandler(logging.StreamHandler(file("acme_motd.log", "w")))
+logger.addHandler(logging.StreamHandler(open("acme_motd.log", "w")))
```

Comment:
Should we use `logging.FileHandler` here? We should set encoding too in any case.
(Really the file also needs to be closed when the program exits... but that's an orthogonal issue.)

Reply:
I think `logging.FileHandler` already has machinery to close the file at process exit (via an `atexit` handler).